README.md

```diff
@@ -1,13 +1,11 @@
 # Project Name
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week project was to create the Guess Who game
```

Comment:
The readme could have some more text on code features. Since you have such a nice written code, please let the readers know that!!!

Reply:
Thank you Mamite! You are right!